### DISCUSSION TITLE
hidden class can not be overwritten by lg:block

### DISCUSSION THREAD

**Discussion by lethanhvietctt5** [tailwindlabs/tailwindcss#5001] [Help] [answered]
### What version of Tailwind CSS are you using?

2.2.4

### What build tool (or framework if it abstracts the build tool) are you using?

Next.js 11.0.1

### What version of Node.js are you using?

v14.17.3

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### Reproduction repository

https://github.com/lethanhvietctt5/react-spotify

### Describe your issue

I have a project that clone Spotify and I'm having an issue with TailwindCSS on Chrome browser, class **``hidden``** can't be overwritten by **``lg:block``**  although the width of the screen was over 1024px.

- Here is the codes:

![image](https://user-images.githubusercontent.com/45467177/126039394-55ba75ef-3a0e-4d91-8a94-7f0feda375c7.png)

- And here is the result CSS in Chrome:

![image](https://user-images.githubusercontent.com/45467177/126039413-6f2147a8-38ee-4c04-b7b2-92623854dc8f.png)

I tried other browsers like Edge, Firefox and it was worked perfectly except for Chrome

**Comment by JimLovesPineapples** [comment]
Thanks for posting this , I was trying to figure out why `className='hidden md:block'` was not working. Also working on other browsers except Chrome

**Reply by lethanhvietctt5** [reply]
Let check your extensions, some classes of them have been auto-added to CSS codes

**Comment by adamwathan** [comment, marked as answer]
Can someone provide a minimal reproduction? I cloned down this project to try it out but it just redirect to Spotify as soon as I start it and I can't afford the time to go down the rabbit hole of figuring out all this unrelated setup stuff unfortunately.

I can't reproduce in Chrome with the most basic example here, and we definitely don't generated `hidden` _after_ the responsive classes in the stylesheet:

https://user-images.githubusercontent.com/4323180/126047202-ecbc4f74-8eb4-47fe-8840-ed378c8e4ad0.mov

https://play.tailwindcss.com/DmwUSDCOtl?size=698x720

We certainly don't generate the `.TnITTtw-hidden` class so I wonder if there is some other CSS involved somehow.

This is the CSS generated by your project when running `npm run build`, and you can see `hidden` only appears once and it's in the correct place:

https://gist.github.com/adamwathan/3908b49694aacffdd832d83b202226c5

Maybe you have a Chrome extension installed that is adding CSS for that class? Going to move to a discussion in case others have ideas but this doesn't appear to be an issue in Tailwind itself.

**Reply by lethanhvietctt5** [reply]
Thanks ! I have checked the CSS code and found some extensions have auto-added some classes to CSS code. Removing them help me to solve this.

**Comment by yayano** [comment]
Thanks, it was dark mode extension i remove it to solve the issue
